OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -819,6 +836,8 @@ public interface IPathFinder
 		/// Only terrain is taken into account, i.e. as if <see cref="BlockedByActor.None"/> was given.
 		/// This would apply for any actor using the given <see cref="Locomotor"/>.
 		/// </summary>
+		/// <remarks>Path searches are not guaranteed to by symmetric,
+		/// the source and target locations cannot be swapped.</remarks>
```

Comment:
Perhaps clarify. Reference this pull request perhaps. 

/// Different conditions and validtions apply to source or target. Swapping them may result in no or an invalid path.

Reply:
The conrete class has a comment with that level of detail: https://github.com/OpenRA/OpenRA/pull/20767/files#diff-db3cc16c7cd82d69f82b046cb747eedb71589480544257e2964ea9b94c2a2e65R107-R110.

But I have left the interface intentionally more general as different pathfinder implementation may choose to operate differently. It may have symmetric path searches for example.